backend/constants/constants.json

```diff
@@ -6,7 +6,9 @@
     "CRITICAL": "Critical Error",
     "WARNING": "Warning",
     "USER": "User ",
-    "UNEXPECTED": "Unexpected Error"
+    "UNEXPECTED": "Unexpected Error",
+    "REQEMPTYFIELDS":"Required Fields Not Filled"
```

Comment:
We won't need this variable after implementing the above commented approach. So, you can get rid of it.